paper-slider.html

```diff
@@ -474,7 +474,7 @@
       ready: function() {
         // issue polymer/polymer#1305
         this.async(function() {
-          this._updateKnob(this.value);
+          this._updateKnob(this.value, this.min, this.max);
         }, 1);
```

Comment:
Afaict, the comment refers to timing issue when using `async with 0 (or no time) and 1`, not if the `_updateKnob` call is actually needed at all.